docs/usage.md

```diff
@@ -62,10 +62,61 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Handling git branches
+
+Use `git-branch` command to work on branch related cases on base repo and any enabled modules.
+By default, a git checkout action will be performed to the revision
+in the base repository. However, the `git-branch --operation` option can be provided to change this
+behavior.
+
+The option takes one of the following as an argument:
+
+* **checkout** [default] - Perform a `git checkout` to checkout the found revision.
+* **rebase** - Perform a `git rebase` to rebase changes on top of the found revision.
+* **merge** - Perform a `git merge` to merge changes up to the found revision into the current branch.
+
+### Checkout branches
+To create a new branch or checkout to a certain revision. The revision to be checked out would be the
+revision for the module in the evergreen manifest associated with the latest commit in the history that
+was run in evergreen.
+
+For the **checkout** option, you can specify a branch name to create with the `-b` or
+`--branch` option.
+
+#### Examples
+To create a branch named `my-branch`, use the following:
+
+```bash
+evg-module-manager git-branch -o checkout --branch my-branch
+```
+
+Checkout to a certain revision, you can issue the following:
+```bash
+evg-module-manager git-branch -o checkout "revision_to_checkout"
+```
+
+### Rebase/Merge branches
+To update the branch with the latest comment on all enabled modules.
+
+For the **rebase** and **merge** operations, if any merge conflicts occur, they will be reported and
+the repository will be left in the unmerged state for manually resolution.
+
+#### Examples
+To merge my active branch on the most recent comment:
+
+```bash
+evg-module-manager git-branch --operation=merge
```

Comment:
In these next to examples, you have switch from using a space between the option and the option value to using an `=` between them. I think it would be keep to use a consistent style.

Reply:
Please remove all the `=` style options. It is not used in other places in the document.